client/src/components/home/Home.tsx

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { DefaultButton, Icon, IContextualMenuProps, Link, PartialTheme, StackItem, Theme } from '@fluentui/react';
+import { Stack, Text, Image } from '@fluentui/react';
+import imageCalendar from '../../assets/lightCalendarSymbol.png';
+import {
+  btnStackStyles,
+  buttonStyle,
+  containerStyles,
+  font12pxSemiBoldStyle,
+  font12pxWeight400Style,
+  font16pxStyle,
+  fullScreenStyles,
+  innerContainer,
+  lineHeight22px,
+  lineHeight28px,
+  linkStyles,
+  newWindowIconWrapper,
+  textDecorationNone
+} from '../../styles/Home.styles';
+
+export interface HomeProps {
+  companyName: string;
+  theme?: PartialTheme | Theme;
+}
+
+const menuProps: IContextualMenuProps = {
+  items: [
+    {
+      key: 'host',
+      text: 'as host (presenter)'
+    },
+    {
+      key: 'attendee',
+      text: 'as attendee'
+    }
+  ]
+  // By default, the menu will be focused when it opens. Uncomment the next line to prevent this.
+  // shouldFocusOnMount: false
+};
+
+export const HomeComponent = (props: HomeProps): JSX.Element => {
+  return (
+    <Stack styles={fullScreenStyles}>
+      <Stack
+        horizontalAlign="center"
+        verticalAlign="start"
+        styles={containerStyles(props.theme)}
+        tokens={{ childrenGap: 15 }}
+      >
+        <Stack styles={innerContainer}>
+          <Stack verticalAlign="center" horizontalAlign="center">
+            <Image width="23.75rem" height="6.25rem" src={imageCalendar} alt="calendarImage"></Image>
+          </Stack>
+          <Stack>
+            <Text styles={lineHeight28px}>Hello,</Text>
+            <Text styles={lineHeight22px}>What would you like to do?</Text>
+            <Stack horizontal horizontalAlign="space-between" styles={btnStackStyles}>
+              <StackItem>
+                <DefaultButton text="Book an appointment" styles={buttonStyle} iconProps={{ iconName: 'Calendar' }} />
+              </StackItem>
+              <StackItem>
+                <DefaultButton
+                  iconProps={{ iconName: 'Video' }}
+                  text="Start a call"
+                  split
+                  splitButtonAriaLabel="See 2 options"
+                  aria-roledescription="split button"
+                  menuProps={menuProps}
+                  styles={buttonStyle}
+                />
+              </StackItem>
+              <StackItem>
+                <DefaultButton text="Join from link" styles={buttonStyle} iconProps={{ iconName: 'Link' }} />
+              </StackItem>
+            </Stack>
+            <Text styles={font16pxStyle}>Frequently asked questions</Text>
```

Comment:
This component has become pretty large and difficult to follow. In general, it is better to have smaller components and compose them together, rather than have one big component.

Reply:
Lets have the "frequently asked questions" as a separate component as this is used in /book with empty bookings link as well

Reply:
As per our offline discussion, this has been done so that it can be reused in @tejaswip-msft PR. Note that if in future the 2 pages' FAQs or purposes diverge we must be careful while making changes